app/views/transactions.html

```diff
@@ -105,7 +94,7 @@ <h3 style="margin: 20px;" ng-if="transactionsSelected.length < defaultBulkRerunL
                 </tr>
               </thead>
               <tbody>
-                <tr class="table-list" ng-repeat="transaction in transactions" ng-class="{transactionsNew: baseIndex + $index < 0}" ng-click='viewTransactionDetails("transactions/" + transaction._id, $event)'>
+                <tr class="table-list" ng-repeat="transaction in transactions|limitTo:settings.filter.limit" ng-class="{transactionsNew: baseIndex + $index < 0}" ng-click='viewTransactionDetails("transactions/" + transaction._id, $event)'>
```

Comment:
This limit breaks the 'Show More' functionality.

Reply:
Good catch